js-miniapp-bridge/src/types/custom-permissions.ts

```diff
@@ -6,6 +6,7 @@ export enum CustomPermissionName {
   LOCATION = 'rakuten.miniapp.device.LOCATION',
   SEND_MESSAGE = 'rakuten.miniapp.user.action.SEND_MESSAGE',
   POINTS = 'rakuten.miniapp.user.POINTS',
+  ATTACHMENTS = 'rakuten.miniapp.device.FILE_DOWNLOAD',
```

Comment:
I think we should also change the name to match:
```suggestion
  FILE_DOWNLOAD = 'rakuten.miniapp.device.FILE_DOWNLOAD',
```

Reply:
sure, fixed that 👍